Web3Unity/Scripts/Prefabs/EVM/AllErc721Example.cs

```diff
@@ -19,6 +19,8 @@ async void Start()
         string network = "rinkeby"; // mainnet ropsten kovan rinkeby goerli
         string account = "0xebc0e6232fb9d494060acf580105108444f7c696";
         string contract = "";
+        int first = 500;
+        int skip = 0;
         string response = await EVM.AllErc721(chain, network, account, contract);
```

Comment:
@KBryan you need to add `first` and `skip` into the params

`string response = await EVM.AllErc1155(chain, network, account, contract, first, skip);`